app/controllers/Tasks_controller.rb

```diff
@@ -0,0 +1,86 @@
+class TasksController < ApplicationController
```

Comment:
This controller is named correctly, but weirdly enough, I'd expect your file name to be `tasks_controller.rb`, not capitalized like how it currently is (`Tasks_controller`)